packages/tideline/js/pool.js

```diff
@@ -315,7 +319,7 @@ function Pool(container) {
   };
 
   /**
-   * @param {{ type: string, name?: string, dataFill?: boolean, pan?: boolean }} plotOptions
+   * @param {{ type: string; name?: string; dataFill?: boolean; pan?: boolean;}} plotOptions
```

Comment:
why changing `,` into `;` ? It's an object, needs `,`

Reply:
It's a Typescript definition of an interface.
And it use ";" for separator normally.